src/components/InfoBox.tsx

```diff
@@ -4,7 +4,7 @@ interface InfoBoxProps {
 const InfoBox: React.FC<InfoBoxProps> = ({ text }) => {
   return (
     <div className="my-8 w-11/12 px-7 text-left">
-      <div className="font-rsd-alt bg-rsd-dark-blue rounded-4xl px-20 py-20 text-2xl font-bold text-white">
+      <div className="font-rsd-alt bg-rsd-dark-blue mid:text-2xl mid:px-20 mid:py-20 rounded-4xl px-10 py-10 text-lg font-bold text-white sm:px-12 sm:py-12 sm:text-xl xl:text-3xl">
```

Comment:
typo here is messing up some styling on larger sizes
```suggestion
      <div className="font-rsd-alt bg-rsd-dark-blue md:text-2xl md:px-20 md:py-20 rounded-4xl px-10 py-10 text-lg font-bold text-white sm:px-12 sm:py-12 sm:text-xl xl:text-3xl">
```